conflict_resolution.html

```diff
@@ -0,0 +1,200 @@
+                <li>Continue to safely work and contribute to the W3C after an issue has been raised</li>
+            </ul>
+                
+            <p>The Code of Conduct is the standard that all W3C participants are required to follow and is used as the basis for any resolution and disciplinary actions. In exceptional circumstances, individuals may have their W3C participation suspended or withdrawn as a result of a complaint or investigation.</p>
+                
+            <p>All situations are different and W3C participants are encouraged to use whatever method of conflict resolution they feel comfortable with.</p>
+                
+            <p>Participants are encouraged to try and resolve issues themselves in the first instance. However, if participants do not feel comfortable or safe doing so, they can get help through an ombudsperson, facilitators and/or mediation.</p>
+                
+            <p>Depending on the severity of the situation, it may not be appropriate to attempt to resolve the issue informally or in the context of the group. Participants should immediately report an incident whenever they feel unsafe or threatened by the behaviour or actions of other W3C participants.</p>
+                
+            <p>Everyone involved in the conflict resolution process is expected to:</p>
+            
+            <ul>
+                <li>Show respect for others</li>
+                <li>Work together to resolve the complaint</li>
+                <li>Maintain confidentiality</li>
+            </ul>
+                
+            <p>A disciplinary investigation can be started where one of the following apply:</p>
+                
+            <ul>
+                <li>It’s not appropriate to begin with mediation</li> 
+                <li>All other mediation or resolution attempts have not worked</li>
+            </ul>
+        </section>
+
+        <section id="procedures">
+            <h2>Conflict Resolution Procedures</h2>
+
+            <p>When an incident arises it is important for everyone involved to know what will happen and what resources are available to them. Any time a Code of Conduct violation or complaint occurs it is important that those involved, especially the party/parties affected by the incident, feel supported and respected.</p>
+
+            <p>How a conflict is handled will depend on a number of factors including the type of complaint, severity, frequency, and contributing factors (for example, has anyone been involved in incidents before, is there a pattern?).</p>
+
+            <p>The procedure for addressing a conflict will include some or all of the following steps:</p>
+
+            <ol>
+                <li>Incident is reported.
+                    <ul>
+                        <li>Reporting can take on many forms including reporting the issue to a chair, ombudsperson, through an anonymous report, or a member of the team.</li>
```

Comment:
Somewhere I think it would be good to provide guidance about when to report an incident, and encourage reporting.  Not every incident needs to be reported, but it is helpful if incidents are reported even when no further action is expected, in order to help detect patterns and repeat offenders.

Reply:
@dbooth-boston, I completely agree that it's important for the community to know that they can report any incidents. Additionally, it's crucial that the management team at W3C regularly reviews the analytics and makes them transparent to both the community and the board. By doing so, we can ensure that the PWE is given the priority it deserves instead of being treated as an afterthought.

Reply:
@wareid @dbooth-boston @TzviyaSiegman I suggest creating sections on Incident Reporting Guidance, Report Management oversights, and Analytics Transparency for the W3C members.
- ### Guidance to Reporting an Incident.
- ### Routine W3M Management Review of Incident Reporting
- ### Transparency of Analytics to W3C Community

Individuals' privacy and confidentiality can be protected by sharing analytics instead of actual reports.

Reply:
I don't think it's a good idea to create guidance for WHEN to report an incident, because there are things that might SEEM unimportant but do actually merit reporting. In terms of analytics and incident reporting metrics, this will be covered by ombuds training. We need to work with the Team to assess who it is exposed to.